app/pb_kits/playbook/pb_checkbox/_checkbox.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= content_tag(:label, id: object.id, data: object.data, value: object.value, name: object.name, class: object.classname(object.kit_class)) do %>
+
+<input type="checkbox" name="<%=object.name%>" <%=object.checked%> ></input>
+<span class="pb_checkbox_checkmark">
+<%= pb_rails("icon", props: { icon: "check",id:"check_icon"}) %></span>
```

Comment:
I would suggest creating the icon through the kit class.  See example in [Icon Circle](https://github.com/powerhome/playbook/blob/master/app/pb_kits/playbook/pb_icon_circle/icon_circle.rb#L29)

Reply:
Does this give an advantage over this method @markiearnold  ?

Reply:
@jasoncypret I wasn't sure if it would effect performance since it had to go through the pb_rails helper logic rather than using the kit class directly. Not sure if that is an actual concern we should worry about. The only other reason for the suggestion was that all other kits are referencing other kits from within the Class, and it would be consistent.